content/700-optimize/200-getting-started.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Getting Started'
+metaTitle: 'Getting started with Prisma Optimize'
+metaDescription: 'Learn how to quickly set up and start using Prisma Optimize.'
+tocDepth: 3
+toc: true
+---
+
+## Prerequisites
+
+Before you begin with Prisma Optimize, ensure you have the following:
+
+- A [Prisma Data Platform account](https://console.prisma.io).
+- A project using [Prisma Client](/orm/prisma-client) version `5.0.0` or higher (we recommend using the latest version).
+- A PostgreSQL, MySQL/MariaDB, PlanetScale, CockroachDB, MS SQL Server, or MongoDB database.
+
+## 1. Launch Optimize
+
+1. Log in to your [Prisma Data Platform account](https://console.prisma.io/optimize).
```

Comment:
add UTM params. Is https://console.prisma.io/optimize a redirect? in which UTMs might get stripped

Reply:
```suggestion
1. Log in to your [Prisma Data Platform account](https://console.prisma.io/optimize?utm_source=docs&utm_medium=optimize-page).
```